README.md

```
@@ -286,10 +287,63 @@ When using the "AI Input" feature:
+Optional. The OAuth metadata auto-derives the public origin from `VERCEL_URL` or `X-Forwarded-*` headers, so a standard Vercel deploy works out of the box. Set this only when running behind a proxy that doesn't forward those headers, or to pin the issuer to a specific domain:
+
+```bash
+# .env.local (or Vercel env var) — no trailing slash
+MCP_PUBLIC_URL=https://your-app.example.com
+```
+
+### Connecting in claude.ai
+
+1. In Claude, open **Settings → Connectors → Add custom connector**.
+2. Paste the MCP endpoint URL:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)